components/infobox/wikis/rocketleague/infobox_person_player_custom.lua

```diff
@@ -52,29 +52,36 @@ function CustomPlayer.run(frame)
 	return player:createInfobox(frame)
 end
 
+function CustomInjector:_parseActive(yearsActive, get, getArg)
+	if String.isEmpty(yearsActive) then
+		return get(getArg)
+	elseif yearsActive == 'hide' then
+		return nil
+	else
+		return Page.makeInternalLink({onlyIfExists = true}, yearsActive)
```

Comment:
Why only create link for manual input, and not automatic (or the other way around)?

Reply:
This is from the original code, I don't know what an internal link is

Reply:
`[[FooBar]]`